### PR TITLE
Fix/keybroker race on key

### PR DESCRIPTION
**Pull request by zdjones**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,10 +3,12 @@ module github.com/LUSHDigital/core
 go 1.14
 
 require (
+	github.com/LUSHDigital/core-lush v0.3.1
 	github.com/LUSHDigital/uuid v0.2.0
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/dustin/go-humanize v1.0.0
 	github.com/golang/protobuf v1.3.2
+	github.com/gorilla/mux v1.7.4
```
Comment (ladydascalie) [inline go.mod]: Why are we importing this?

File: go.mod [hunk 2]
```diff
@@ -3,10 +3,12 @@ module github.com/LUSHDigital/core
 go 1.14
 
 require (
+	github.com/LUSHDigital/core-lush v0.3.1
```
Comment (ladydascalie) [inline go.mod]: Same for this, where are we using core-lush?

**Review by ladydascalie — CHANGES_REQUESTED** [review]
Seems like you need to run a `go mod tidy` but appart from that, great stuff!

File: go.mod (hunk 1 above)
Comment (zdjones) [reply go.mod]: I don't even understand why these changes are here, let me go mod tidy and push again

Comment (zdjones) [reply go.mod]: pushed new patch to remove this

File: go.mod (hunk 2 above)
Comment (zdjones) [reply go.mod]: pushed new patch to remove this

**Review by zeevallin — APPROVED** [review]

**Pull request merged**